src/main/java/com/flab/pricesearch/user/domain/User.java

```diff
@@ -0,0 +1,25 @@
+package com.flab.pricesearch.user.domain;
+
+import lombok.Data;
+
+@Data
+public class User {
+  private String userId;
+  private String password;
+  private String platformType;
+  private String token;
+  private String userName;
+  private String birthDate;
+  private String gender;
+  private String tel;
+  private String email;
+  private String address;
+  private String regDate;
```

Comment:
약어가 아닌 풀 네임으로 가면 좋겠습니다.

```suggestion
  private String registrationDate;
```

그리고 이거 타입이 `String` 인 것이 괜찮을까요?